build.sbt

```diff
@@ -19,19 +19,19 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 lazy val commonSettings = Seq(
   scalafmtOnCompile := true,
   scalacOptions ++= Seq(
-      "-deprecation",
-      "-encoding",
-      "UTF-8",
-      "-feature",
-      "-language:_",
-      "-Xfatal-warnings",
-      "-Xlog-reflective-calls",
-      "-Xlint:_",
-      "-Ybackend-parallelism",
-      "8",
-      "-Ywarn-dead-code",
-      "-Ywarn-unused:-imports,_",
-      "-unchecked"),
+    "-deprecation",
+    "-encoding",
+    "UTF-8",
+    "-feature",
+    "-language:_",
+    "-Xlog-reflective-calls",
+    "-Xlint:_",
+    "-Ybackend-parallelism",
+    "8",
+    "-Ywarn-dead-code",
+    "-Ywarn-unused:-imports,_",
+    "-unchecked",
+    if (sys.env.getOrElse("CI", "false") == "true") "-Xfatal-warnings" else ""),
```

Comment:
Hmm aren't there any problems from an empty-string argument?

Reply:
I tested locally with `sbt clean compile test` and it exited with `0`